examples/integration/xcodeproj.bazelrc

```diff
@@ -3,3 +3,6 @@ import %workspace%/../../xcodeproj.bazelrc
 
 # Disabling archived bundles for fixture coverage
 build:rules_xcodeproj --define=apple.experimental.tree_artifact_outputs=1
+
+build:rules_xcodeproj --apple_generate_dsym
+build:rules_xcodeproj --output_groups=+dsyms
```

Comment:
for testing

Reply:
In the end, all you will need is `build:rules_xcodeproj --apple_generate_dsym`, only in this file.